src/server/game/DataStores/DBCEnums.h

```diff
@@ -168,6 +168,16 @@ enum ArtifactPowerFlag : uint8
 
 #define BATTLE_PET_SPECIES_MAX_ID 2796
 
+enum BattlemasterListFlags
+{
+    BATTLEMASTER_LIST_FLAG_DISABLED             = 0x01,
+    BATTLEMASTER_LIST_FLAG_UNK02                = 0x02,
```

Comment:
Seems like this flag allows you to queue without selected roles?

Reply:
This is correct